launch_gz/launch_gz/actions/gzserver.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2024 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the GzServer action."""
+
+from typing import List
+from typing import Optional
+
+from launch.action import Action
+from launch.actions import IncludeLaunchDescription
+from launch.frontend import expose_action, Entity, Parser
+from launch.launch_context import LaunchContext
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.some_substitutions_type import SomeSubstitutionsType
+from launch.substitutions import PathJoinSubstitution
+from launch_ros.substitutions import FindPackageShare
+
+@expose_action('gzserver')
```

Comment:
This might be bike shedding, but I saw we're using `gz_spawn_model` in https://github.com/gazebosim/ros_gz/pull/551/files#diff-7239435060c9f1cf951f61f1d6645f482a8de8c36c4e8ad20e8a60ec0c6c2539R36-R47. What do you think of making this `gz_server`? We'd have to change the names of the `gzserver.launch` files as well.

Reply:
Renamed in [01345e2](https://github.com/gazebosim/ros_gz/pull/528/commits/01345e2d95d76998f3ab8fbd341e2164d075a009).